tests/cpp/test_runtime.cpp

```diff
@@ -20,7 +21,11 @@ namespace nvfuser {
 
 using RuntimeTest = NVFuserTest;
 
-TEST_F(RuntimeTest, FusionClearGmemBetweenSegments_CUDA) {
+TEST_F(RuntimeTest, ClearGmemBetweenSegments) {
+  at::cuda::clearCublasWorkspaces();
+  releaseZeroedMemory();
+  ASSERT_EQ(memoryAllocated(0), 0) << "Memory leak detected";
```

Comment:
We can also do this in TearDown of all tests so we can verify none of them leak GPU memory. I'm happy to pursue that in a separate PR because the blast radius is larger.